islandora_audio.module

```diff
@@ -236,3 +236,35 @@ function islandora_audio_file_mimetype_mapping_alter(&$mapping) {
   unset($mapping['extensions']['mp3']);
   $mapping['extensions']['mp3'] = $code;
 }
+
+/**
+ * Implements hook_help().
+ */
+function islandora_audio_help($route) {
+  switch ($route) {
+    // Main help page for the module.
+    case 'help.page.islandora_audio':
+      // About section.
+      $output = '<h3>' . t('About') . '</h3>';
+      $output .= '<p>' . t('The Islandora Audio module is one of the many Islandora Solution Packs. This module allows for the creation of audio collections which can be populated by uploading WAV and MP3 files as objects within Islandora. An audio file can then be made available for download by users when viewing the corresponding object. See the <a href=":wiki">Audio Solution Pack wiki*</a> for more information.', [
+        ':wiki' => 'https://wiki.duraspace.org/display/ISLANDORA/Audio+Solution+Pack',
+      ]) .
+      '</p>';
+      // Uses section.
+      $output .= '<h3>' . t('Uses') . '</h3>';
+      $output .= '<dt>' . t('Upload Audio') . '</dt><dd>';
+      $output .= t('At www.yoursite.com/islandora/object/{ID}audio_collection/manage (ex: www.something.com/islandora/object/islandora%3audio_collection/manage), you can click "Add an object to this Collection" and follow the ingest form to upload an audio file in the accepted formats.') .
+      '</dd>';
+      $output .= '<dt>' . t('View Audio Object') . '</dt><dd>' . t('At www.yoursite.com/islandora/object/{ID}, (ex: www.something.com/islandora/object/islandora%3A10), you can play the audio, provided that an appropriate viewer has been configured at www.yoursite.com/admin/config/islandora/solution_pack_config/audio.') .
```

Comment:
Should have one convention, not so sure about mixing `yoursite` and `something`.